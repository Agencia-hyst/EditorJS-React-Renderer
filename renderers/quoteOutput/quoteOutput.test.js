import QuoteOutput from './index';

describe('QuoteOutput renderer:', () => {
	// @TODO
});
