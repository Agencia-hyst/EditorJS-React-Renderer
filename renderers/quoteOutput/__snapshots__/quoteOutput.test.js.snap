// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteOutput renderer: should match snapshot 1`] = `""`;
